src/components/TransactionTable/TransactionTable.js

```diff
@@ -0,0 +1,214 @@
+import React, { PureComponent } from 'react';
+import propTypes from 'prop-types';
+import moment from 'moment';
+import { Table } from '../index';
+import { connectTheme } from '../../utils';
+
+import TransactionView from './TransactionView';
+
+const constants = {
```

Comment:
Would it make sense to have this be able to be overwritten or merged w/ custom text via props?